starknet_py/utils/schema.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from marshmallow import EXCLUDE, RAISE
+from marshmallow import Schema as MarshmallowSchema
+
+
+class Schema(MarshmallowSchema):
+    class Meta:
+        unknown = (
+            EXCLUDE
+            if os.environ.get("STARKNET_PY_MARSHMALLOW_UKNOWN_EXCLUDE")
```

Comment:
> This can be achieved by setting the environment variable, STARKNET_PY_MARSHMALLOW_UKNOWN_EXCLUDE,
to true. Consequently, any unknown fields in response will be excluded.

does this code checks this? I'm not sure, but I think it may just check if the env variable is set (so eg `STARKNET_PY_MARSHMALLOW_UKNOWN_EXCLUDE=false` may also set it, resulting in turning on the exclusion). let's double check.